pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -184,7 +185,27 @@ func (d *kubeDockerClient) InspectImage(image string) (*dockertypes.ImageInspect
 		}
 		return nil, err
 	}
-	return &resp, nil
+	parts := strings.SplitN(image, ":", 2)
```

Comment:
Could probably reuse/share the logic in `applyDefaultImageTag`
https://github.com/kubernetes/kubernetes/blob/9f1c3a4c562a1b5ab96a4c43d95c5c39d80fbec5/pkg/kubelet/dockertools/docker.go#L161